deployability/modules/workflow_engine/examples/manager/vagrant/dtt1-managers-poc-vagrant.yaml

```diff
@@ -2,18 +2,8 @@ version: 0.1
 description: This workflow is used to test manager deployment for DDT1 PoC
 variables:
   manager-os:
-    - linux-ubuntu-20.04-amd64
-    - linux-ubuntu-22.04-amd64
-    - linux-oracle-9-amd64
-    - linux-amazon-2-amd64
-    - linux-redhat-7-amd64
-    - linux-redhat-8-amd64
-    - linux-redhat-9-amd64
     - linux-centos-7-amd64
     - linux-centos-8-amd64
-    - linux-debian-10-amd64
-    - linux-debian-11-amd64
-    - linux-debian-12-amd64
```

Comment:
Restore them

Reply:
Done in [a7aa606](https://github.com/wazuh/wazuh-qa/pull/5392/commits/a7aa6062b7d70d81d64975f698f29e7d4959c353)